### PR TITLE
Fix update script's failure to update netbird-ui in binary installation

### PR DESCRIPTION
**Pull request by bcmmbaga**
## Describe your changes

Resolve the problem with the update script that prevents netbird-ui from updating during binary installation.

Introduce the variable UPDATE_NETBIRD. Now we can upgrade the binary installation with:
```
export UPDATE_NETBIRD=true
curl -fsSL https://pkgs.netbird.io/install.sh | sh
```

### Checklist
- [x] Is it a bug fix
- [ ] Is a typo/documentation fix
- [ ] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by braginini** [comment]
NetBird UI doesn't stop when upgrade happens. Therefore, the currently running version is not updated and the user needs to restart it. Is this the expected behaviour? Should we send a stop signal?

**Comment by bcmmbaga** [comment]
> NetBird UI doesn't stop when upgrade happens. Therefore, the currently running version is not updated and the user needs to restart it. Is this the expected behaviour? Should we send a stop signal?

Yes, for now, this is the expected behavior. The script is designed to only stop the NetBird service during upgrades. However, if it's necessary to close the UI as well, we can send a signal to do so

File: release_files/install.sh [hunk 1]
```diff
@@ -284,14 +284,21 @@ is_bin_package_manager() {
   fi
 }
 
+stop_running_netbird_ui() {
+  if pgrep -x "netbird-ui" >/dev/null; then
```
Comment (mlsmaycon) [inline release_files/install.sh]: is pgrep on most devices?

File: release_files/install.sh [hunk 2]
```diff
@@ -284,14 +284,21 @@ is_bin_package_manager() {
...
+    echo "NetBird UI is running. Stopping it..."
+    pkill -x "netbird-ui"
```
Comment (mlsmaycon) [inline release_files/install.sh]: same for pkill, let's use `ps ` and `grep` combination?

**Review by mlsmaycon — APPROVED** [review]

**Pull request merged**